drivers/performance_metrics/app/models/performance_metrics/report.rb

```diff
@@ -676,20 +682,15 @@ def report_labels
 
     private def run_spm
       # puts 'Running SPM'
-      # Looking for SPM Measure 1A E3
-      # Looking for SPM Measure 2 J7 (total returns to homelessness within 2 years)
-      questions = [
-        'Measure 1',
-        'Measure 2',
-      ]
+      questions = ['Measure 2']
```

Comment:
doesn't seem like measure 1 is used, comment seems out of date

Reply:
Totally possible.